flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -217,6 +217,7 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         this.onRightFunction = (f) => this.onLsk(this.onRightInput[f], this.rightInputDelay[f]);
         this.onOvfy = () => this.scratchpad.addChar('Δ');
         this.onUnload = () => {};
+        this.onAirport = () => {};
```

Comment:
if using an arrow function, then it should be declared in `FMCMainDisplay` constructor and initialized by `initVariables()`

Reply:
Ah, I was wondering if that was the right way to do it. Should be fixed now. Let me know if I need to change something else 👍.